win32ss/user/ntuser/message.c

```diff
@@ -1433,11 +1433,17 @@ co_IntSendMessageTimeoutSingle( HWND hWnd,
         RETURN( TRUE);
     }
 
-    if (uFlags & SMTO_ABORTIFHUNG && MsqIsHung(ptiSendTo))
+    if (MsqIsHung(ptiSendTo))
```

Comment:
Did you see https://github.com/reactos/reactos/blob/master/win32ss/user/ntuser/message.c#L1470 ?